docs/Manual.md

```diff
@@ -273,7 +278,11 @@ http:
   # A Website
   - name: MegaEase Website (Global)
     url: https://megaease.com
-
+    labels:
+      - name: team
+        value: ease
+      - name: owner
+        value: megaease
```

Comment:
as we merged all constant labels into one set, using a map instead of an array will be more user friendly.

```suggestion
    labels:
      team: ease
      owner: megaease
```

Reply:
updated